Sources/Plasma/FeatureLib/pfDXPipeline/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+set(pfDXPipeline_SOURCES
+    hsGDirect3D.cpp
+    plDXDevice.cpp
+    plDXDeviceRefs.cpp
+    plDXEnumerate.cpp
+    plDXPipeline.cpp
+    plDXPixelShader.cpp
+    plDXShader.cpp
+    plDXTextFont.cpp
+    plDXVertexShader.cpp
+)
+
+set(pfDXPipeline_HEADERS
+    hsGDirect3D.h
+    plDXBufferRefs.h
+    plDXDevice.h
+    plDXDeviceRef.h
+    plDXEnumerate.h
+    plDXLightRef.h
+    plDXPipeline.h
+    plDXPixelShader.h
+    plDXRenderTargetRef.h
+    plDXSettings.h
+    plDXShader.h
+    plDXTextFont.h
+    plDXTextureRef.h
+    plDXVertexShader.h
+)
+
+plasma_library(pfDXPipeline
+    SOURCES ${pfDXPipeline_SOURCES} ${pfDXPipeline_HEADERS}
+    PRECOMPILED_HEADERS Pch.h
```

Comment:
This project's Pch.h should probably gain the contents of plPipeline/Pch.h because this file only includes the D3D9 bits.

Reply:
:+1:

And we still want `plPipeline/Pch.h` too?

Reply:
Yes, keep `plPipeline/Pch.h`, but I think just copy the content into this file as well. We may tweak the contents at some point so we aren't compiling anything unnecessary. (But, in my build profiling, PCHs never slow the build down...)